locksmithctl/daemon.go

```diff
@@ -297,10 +300,27 @@ func runDaemon() int {
 	}
 
 	if strategy == StrategyOff {
-		fmt.Fprintf(os.Stderr, "Reboot strategy is %q - shutting down.", strategy)
+		fmt.Fprintf(os.Stderr, "Reboot strategy is %q - shutting down.\n", strategy)
 		return 0
 	}
 
+	startw := os.ExpandEnv("${REBOOT_WINDOW_START}")
+	lengthw := os.ExpandEnv("${REBOOT_WINDOW_LENGTH}")
+	if (startw == "") != (lengthw == "") {
+		fmt.Fprintln(os.Stderr, "both $REBOOT_WINDOW_START and $REBOOT_WINDOW_LENGTH must be specified")
```

Comment:
nit, "either both or neither of REBOOT_WINDOW_START and REBOOT_WINDOW_LENGTH must be set"